### DISCUSSION TITLE
Can you bring back the AI noise canceling feature?

### DISCUSSION THREAD

**Discussion by thanhlocgp** [seerge/g-helper#413] [Ideas]
It so so helpful for those people who need to make a phone call, meeting everyday.

**Comment by seerge** [comment]
@thanhlocgp it's proprietary, and is handled by Fortemedia and Realtek services / drivers.  You may check this topic https://github.com/seerge/g-helper/issues/18 one some models realtek control panel offers some control for it too (but not on all, incl g14)
